src/components/feedback/segment.js

```diff
@@ -0,0 +1,58 @@
+import forwardEvent from './forward-event';
+import * as Sentry from '@sentry/browser';
+import env from '../analytics-shell/env';
+
+// Send the event to Segment with forward event
+export function sendToSegment({ state, props }) {
+  const event = createSegmentEvent({ state, props });
+  forwardEvent(event, props.webhook, (err) => Sentry.captureException(err));
+}
+
+// Creates event to send to Segment
+function createSegmentEvent({ state, props }) {
+  const {
+    user,
+    anonymousId,
+    helpful,
+    feedback,
+    sessionId,
+    category,
+    categoryType,
+    contactSupport
+  } = state;
+  const { site, section, location } = props;
+  // set user if available else set anonymousId (needed for forward-event request)
+  const identity = {
+    ...(user && user.id ? { userId: user.id } : { anonymousId: anonymousId })
+  };
+  return {
+    event: 'Sent docs feedback',
+    ...identity,
```

Comment:
Do we need the `username` and `anonymousId` both here and in line 51?

Reply:
We do! Fixing!

Reply:
Oh! But we only need 1 (userId or anonymousid) for this particular line.